orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/scripts/restore_db.sh

```diff
@@ -65,6 +65,10 @@ export PGSSLMODE=verify-full
 {%- endif %}
 
 errorExit() {
+  if [ -d "$BACKUPS_DIR" ]; then
+      rm -rfv "$BACKUPS_DIR" > >(tee -a $LOGFILE) 2> >(tee -a $LOGFILE >&2)
```

Comment:
It is unsafe to remove everything with `rm -rf`.
We should know what files are copied, right? These are database files and we should delete them only. Same applies to downloading - we better only download what we need to restore.
I know this is how it was done before this way but there is a room for improvement.

Reply:
I still think we better remove one by one. You never know the day someone "optimizes" the script to have this directory configurable and then one day point to "/" . And `rm -rf /` is very dangerous.

Reply:
If so that may bring up another concern - will the name of the temp files always be hive_backup and ranger_backup? Adding in @kkalvagadda1 and @sxxgrc for this topic since we discussed about this long time ago.

Reply:
Hi @votez and @RokolyA94 CC @sxxgrc I updated the code to check if the path starts with the expected path and then remove it. Please let me know if this approach looks good to you. Thank you!!

Reply:
If we are changing the normal behavior of restore I'd rather to open a new jira ticket to keep things simple. Please let me know what you think!=)

Reply:
I'm hearing two questions
1. If we have to delete it by file, we need to make sure files for all the databases are cleaned. There would be more databases we should consider. There will be databases from data catalog service. THis script is also used by DH to backup databases for other services.
2. I'm with @votez on deleting the file after a restore is complete instead of deleting all of them at the end. This could be a separate PR.

Reply:
@lajosrodek We don't have issue for #1 that Kalyan brings up since this change is not deleting it by file but deleting by directory. I created a jira regarding #2, which is an improvement for deleting files: [CDPSDX-4017](https://jira.cloudera.com/browse/CDPSDX-4017)